server/sonar-server/src/test/java/org/sonar/server/user/index/UserIndexTest.java

```diff
@@ -112,4 +112,17 @@ public void get_nullable_by_scm_account_return_null_when_two_users_have_same_ema
     assertThat(index.getNullableByScmAccount("user1@mail.com")).isNull();
   }
 
+  @Test
```

Comment:
should you test when there's more than 3 accounts, that it returns only three accounts ?